Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -122,17 +122,7 @@ open class NavigationMapView: UIView {
     
     var currentLineGradientStops = [Double: UIColor]()
     var currentRestrictedAreasStops = [Double: UIColor]()
-    var routeLineTracksTraversal: Bool = false {
-        didSet {
-            if routeLineTracksTraversal, let route = self.routes?.first {
-                initPrimaryRoutePoints(route: route)
-                setUpLineGradientStops(along: route)
-            } else {
-                removeLineGradientStops()
-            }
-            updateRestrictedAreasGradientStops(along: self.routes?.first)
-        }
-    }
+    var routeLineTracksTraversal: Bool = false
```

Comment:
I'm not sure whether should we add the public methods of `NavigationMapView.turnOnRouteLineTracksTraversal()` and `NavigationMapView.turnOffRouteLineTracksTraversal()`, or just make the `NavigationMapView.routeLineTracksTraversal` a public property.

Reply:
I'd go for a property variant. Proposed methods basically do the same property setup, but are less convenient to use.

Reply:
Thanks for the suggestion, right now it's using the public property.